dags/asn_aggregates.py

```diff
@@ -0,0 +1,48 @@
+from airflow import DAG
+from airflow.operators.sensors import ExternalTaskSensor
+from datetime import timedelta, datetime
+from utils.gcp import bigquery_etl_query
+
+default_args = {
+    "owner": "ascholtz@mozilla.com",
+    "email": ["ascholtz@mozilla.com", "tdsmith@mozilla.com",],
+    "depends_on_past": False,
+    "start_date": datetime(2020, 4, 5),
+    "email_on_failure": True,
+    "email_on_retry": True,
+    "retries": 2,
+    "retry_delay": timedelta(minutes=30),
+}
+
+dag = DAG("asn_aggregates", default_args=default_args, schedule_interval="@daily")
```

Comment:
I prefer the context manager style `with DAG(...):` which avoids needing to specify `dag=dag` on each task which is error prone. I believe we use that style consistently in this repo besides main_summary, which I assume is just a historical artifact.